src/events/guildCreate.ts

```diff
@@ -14,23 +17,22 @@ export default {
 
     async run(guild: Guild) {
       const owner = await guild.fetchOwner();
-      const dmChannel = (await owner.createDM().catch(() => null)) as DMChannel | null;;
+      const dmChannel = (await owner.createDM().catch(() => null)) as DMChannel | null;
       const hearts = ["💖", "💝", "💓", "💗", "💘", "💟", "💕", "💞"];
 
       const embed = new EmbedBuilder()
         .setTitle("👋 • Welcome to Nebula!")
         .setDescription([
           "Nebula is a multiplatform, multipurpose bot with the ability to add extensions to have additional features.",
-          "To do things like disabling/enabling commands, use the **/settings** command.",
-          "As of now, it's in an early stage of development. If you find bugs, please go to our [official server](https://discord.gg/7RdABJhQss) to report them."
+          "To manage the bot, sign into the [dashboard](https://dash.nebulabot.org).",
+          "Nebula is in an early stage of development. If you find bugs, please go to our [official server](https://discord.gg/7RdABJhQss) and report them."
```

Comment:
Dasshubusudo update!! 👀 (I don't know if you spell it like that but yeah)

Reply:
dasshubodo